src/App.js

```diff
@@ -1,25 +1,38 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState, useEffect } from "react";
+import SharedPage from "./pages/SharedPage";
+import { getSampleUser, getSampleFolder } from "./api";
+import "./styles/global.css";
+
+const App = () => {
+  const [user, setUser] = useState({});
+  const [folderObj, setFolderObj] = useState({});
+  const [loadingError, setLoadingError] = useState(null);
+
+  const handleLoad = async () => {
+    let userBody;
+    let folderBody;
+
+    try {
+      setLoadingError(null);
+      userBody = await getSampleUser();
+      folderBody = await getSampleFolder();
+      setUser(userBody);
+      setFolderObj(folderBody);
```

Comment:
```suggestion
    try {
      setLoadingError(null);
      const userBody = await getSampleUser();
      const folderBody = await getSampleFolder();
      setUser(userBody);
      setFolderObj(folderBody);
```
가 더 좋아보이네요. 굳이 let으로 위에 선언하고 값을 변경할 필요는 없어 보입니다.

Reply:
처음에는 let이 필요했었는데 코드를 수정하다보니 필요가 없어졌네요. const로 수정했습니다!